### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/tutorial/dependencies/global-dependencies.md`

### PR DESCRIPTION
**Pull request by osangu**
Add Korean translation for `docs/ko/docs/tutorial/dependencies/global-dependencies.md`

Related to Korean translations https://github.com/tiangolo/fastapi/issues/2017

p.s
> sorry for close and reopening PR.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4966d65d16b98e6944146141d08490f0d0e3fb95 at: https://ebfaf65e.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit b7d5bf61bf88abd8faae8ce71853940e33c8ee99 at: https://0f5e5cf3.fastapitiangolo.pages.dev

File: docs/ko/docs/tutorial/dependencies/global-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ```Python hl_lines="15"
+    {!> ../../../docs_src/dependencies/tutorial012.py!}
+    ```
+
+[*작업 경로 데코레이터*에 `의존성` 추가하기](dependencies-in-path-operation-decorators.md){.internal-link target=_blank} 섹션에 있는 모든 아이디어는
+여전히 적용됩니다. 그러나 위 경우에는, 애플리케이션의 모든 *경로 작동*에 적용됩니다.
+
+## *경로 작동* 집합을 위한 의존성
+
+나중에, 어떻게 여러 개의 파일로 더 큰 애플리케이션을 구조화 하는지에 대해 읽을 때 ([애플리케이션 확장 - 여러 개의 파일](../../tutorial/bigger-applications.md){.internal-link target=_blank}), 어떻게 *작업 경로*의 묶음을 위한 하나의 `의존성` 매개변수를 선언하는지 배우게 될 겁니다.
```
Comment (Sion99) [inline docs/ko/docs/tutorial/dependencies/global-dependencies.md:35]:
```suggestion
나중에, 여러 파일을 사용하여 더 큰 애플리케이션을 구조화하는 방법에 대해 읽을 때 ([애플리케이션 확장 - 여러 개의 파일](../../tutorial/bigger-applications.md){.internal-link target=_blank}), *경로 동작*의 그룹을 위한 단일 `의존성` 매개변수를 선언하는 방법을 배우게 됩니다.
```
According to #3167 , term "Path Operation" seems to be translated to "경로 동작".

File: docs/ko/docs/tutorial/dependencies/global-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
+
+    ```Python hl_lines="15"
+    {!> ../../../docs_src/dependencies/tutorial012.py!}
+    ```
+
+[*작업 경로 데코레이터*에 `의존성` 추가하기](dependencies-in-path-operation-decorators.md){.internal-link target=_blank} 섹션에 있는 모든 아이디어는
+여전히 적용됩니다. 그러나 위 경우에는, 애플리케이션의 모든 *경로 작동*에 적용됩니다.
```
Comment (Sion99) [inline docs/ko/docs/tutorial/dependencies/global-dependencies.md:31]:
```suggestion
[*경로 작동 데코레이터*에 `의존성` 추가하기](dependencies-in-path-operation-decorators.md){.internal-link target=_blank} 섹션에 있는 모든 아이디어는 여전히 적용됩니다. 그러나 위 경우에는, 애플리케이션의 모든 *경로 작동*에 적용됩니다.
```
Seems like there's no need to seperate line 30, 31.

**Pull request closed without merging**